jasmine/src/BowlingGame.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
```

Comment:
Very easy to read code :) Thanks for making my life easier! I like you're using strict as well, shows commitment to writing good syntax